applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -167,8 +167,17 @@ static int set_report(const struct device *dev, struct usb_setup_packet *setup,
 		break;
 
 	case REPORT_TYPE_OUTPUT:
-		if (request_value[0] == REPORT_ID_KEYBOARD_LEDS) {
-			LOG_INF("No action on keyboard LEDs report");
+		if (IS_ENABLED(CONFIG_DESKTOP_HID_REPORT_KEYBOARD_SUPPORT) &&
+		    (request_value[0] == REPORT_ID_KEYBOARD_LEDS)) {
+			struct usb_hid_device *usb_hid = dev_to_hid(dev);
+			struct hid_report_event *event = new_hid_report_event(*len);
```

Comment:
Actually we do not validate len for feature reports too. I will check this one and add validation in both places if needed.

Reply:
Seems like we may actually receive report with size other than defined in the report map. In that case I will ignore it here and print warning since HID reports have fixed size.